src/client/commands/dashboard/top.rs

```diff
@@ -0,0 +1,100 @@
+use crate::client::commands::dashboard::event::{Event, Events};
+use std::cell::RefCell;
+use std::io::Stdout;
+use std::rc::Rc;
+use std::{error::Error, io};
+use termion::raw::RawTerminal;
+use termion::{event::Key, input::MouseTerminal, raw::IntoRawMode, screen::AlternateScreen};
+use tui::{
+    backend::TermionBackend,
+    layout::{Constraint, Direction, Layout},
+    style::{Color, Modifier, Style},
+    widgets::{BarChart, Block, Borders},
+    Terminal,
+};
+
+///Some Random data
+struct AppData<'a> {
+    data_vec: Vec<(&'a str, u64)>,
+}
+
+impl<'a> AppData<'a> {
+    fn new() -> AppData<'a> {
+        AppData {
+            data_vec: vec![
+                ("B1", 9),
+                ("B2", 12),
+                ("B3", 5),
+                ("B4", 8),
+                ("B5", 2),
+                ("B6", 4),
+                ("B7", 5),
+                ("B8", 9),
+                ("B9", 14),
+                ("B10", 15),
+                ("B11", 1),
+            ],
+        }
+    }
+
+    fn update(&mut self) {
+        let value = self.data_vec.pop().unwrap();
```

Comment:
What is this supposed to do?

Reply:
It should compile now, the whole thing was in playground stage. I'll get it to work now that the tako update seems solidified